spec/models/google_analytics/data_tracking_spec.rb

```diff
@@ -2,22 +2,36 @@
 
 module GoogleAnalytics
   describe DataTracking do
+    before do
+      allow(described_class).to receive(:adapter).and_return('Adapter')
+      allow(Rails).to receive(:env).and_return('production'.inquiry)
```

Comment:
👍 ooh, did not know about `inquiry`.

Reply:
I don't really know what it is doing and Rubocop doesn't like it. I just left it in - that line is copied from elsewhere.